_ja/overviews/scala3-migration/compatibility-classpath.md

```diff
@@ -0,0 +1,144 @@
+---
+title: クラスパス互換性
+type: section
+description: このセクションでは、Scala 2.13とScala 3クラスファイルの互換性について説明しています。
+num: 3
+previous-page: compatibility-source
+next-page: compatibility-runtime
+language: ja
+---
+
+コード上で、パブリックな型と構文を利用でき、そしてパブリックメソッドは異なるモジュールやライブラリで定義されています。
+そしてできる限り長く型チェックとして働いて、コンパイルフェーズにてコードの意味的な構成を検証し、クラスファイルに含まれている型や構文やメソッドの重要性を読むことができます。
+
+Scala 2ではシグネチャはpickleフォーマットで保存されます。
+Scala 3では少し違っていて、なぜならシグネチャレイアウトよりも多くのTASTyフォーマットに寄り添っているからです。
```

Comment:
```suggestion
Scala 3 では少し違っていて、シグネチャ情報だけに留まらずその他の多くの情報を持つ TASTy フォーマットを採用している。
```

原文: In Scala 3 the story is a bit different because it relies on the TASTy format which is a lot more than a signature layout.